spec/controllers/profile_controller_spec.rb

```diff
@@ -26,5 +26,24 @@
         expect(response).to_not be_redirect
       end
     end
+
+    context 'when an indentity has been deactivated by password reset' do
+      it 'renders the profile and shows a deactivation banner' do
+        user = create(:user, :signed_up)
+        user.identities << Identity.create(
```

Comment:
I was just copying from the other spec in the file, seems fine with just two instances for now? If you feel strongly, I can switch to a factory